src/interface/src/styleguide/search-bar/search-bar.component.ts

```diff
@@ -88,6 +90,10 @@ export class SearchBarComponent implements OnInit, OnDestroy {
     this.searchInput.next(val);
   }
 
+  onSelectionChange(event: MatAutocompleteSelectedEvent) {
```

Comment:
not a blocker but maybe change the name to something like `onHistoryClick` , `onHistorySelect` or something along those lines

Reply:
Yeah, that's better -- our future selves will thank us